pebble_test.go

```diff
@@ -0,0 +1,51 @@
+//go:build pebbledb
+// +build pebbledb
+
+package db
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPebbleDBBackend(t *testing.T) {
+	name := fmt.Sprintf("test_%x", randStr(12))
+	dir := os.TempDir()
+	db, err := NewDB(name, PebbleDBBackend, dir)
+	require.NoError(t, err)
+	defer cleanupDBDir(dir, name)
+
+	_, ok := db.(*PebbleDB)
+	assert.True(t, ok)
+}
+
+// func TestPebbleDBStats(t *testing.T) {
+// 	name := fmt.Sprintf("test_%x", randStr(12))
+// 	dir := os.TempDir()
+// 	db, err := NewDB(name, PebbleDBBackend, dir)
+// 	require.NoError(t, err)
+// 	defer cleanupDBDir(dir, name)
+
+// 	assert.NotEmpty(t, db.Stats())
+// }
```

Comment:
```suggestion
// func TestPebbleDBStats(t *testing.T) {
// 	name := fmt.Sprintf("test_%x", randStr(12))
// 	dir := os.TempDir()
// 	db, err := NewDB(name, PebbleDBBackend, dir)
// 	require.NoError(t, err)
// 	defer cleanupDBDir(dir, name)

// 	assert.NotEmpty(t, db.Stats())
// }
```
Should we enable this?

Reply:
Just making sure we don't leave comments if they are not needed?